test/acceptance/auto_auth.py

```diff
@@ -80,27 +80,12 @@ def get_user_id(self):
         match = re.search(r' user_id ([^$]+)$', message)
         return match.groups()[0] if match else None
 
-    def get_username(self):
-        """
-        Finds and returns the username of the current user.
-        """
-        username, _ = self._get_username_and_email()
-        return username
-
-    def get_email(self):
-        """
-        Finds and returns the email address of the current user.
-        """
-        _, email = self._get_username_and_email()
-        return email
-
-    def _get_username_and_email(self):
+    def get_username_and_email(self):
```

Comment:
This block was changed in the interest of preventing extraneous bok choy queries (which ended up not solving all our speed problems, but seems reasonable to do anyways)